3-platform/shopping-cart/infra/src/main/java/com/_6core/platform/shopping/cart/infra/adapter/driving/grpc/CreateOrderGrpc.java

```diff
@@ -0,0 +1,9 @@
+package com._6core.platform.shopping.cart.infra.adapter.driving.grpc;
+
+import com._6core.platform.shopping.cart.infra.dto.CreateOrderRequest;
+import com._6core.platform.shopping.cart.infra.dto.CreateOrderResponse;
+import reactor.core.publisher.Mono;
+
+public interface CreateOrderGrpc {
```

Comment:
utilize hexagonal arch terms, e.g. Driving/Driven Adapters, In/Out Ports, DomainService.